pkg/visor/daemon.go

```diff
@@ -0,0 +1,45 @@
+package visor
+
+import (
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/SkycoinProject/skywire-mainnet/pkg/snet"
+
+	"github.com/SkycoinProject/dmsg/cipher"
+	cp "github.com/SkycoinProject/skycoin/src/cipher"
+	"github.com/SkycoinProject/skycoin/src/util/logging"
+	apd "github.com/SkycoinProject/skywire-peering-daemon/src/daemon"
+)
+
+var logger = logging.MustGetLogger("skywire-peering-daemon")
+
+func execute(binPath, publicKey, namedPipe string) error {
```

Comment:
If the `daemon` should be started from the `visor`, maybe it would be more elegant to have its code in `skywire-mainnet`. What do you think?

Reply:
That would work, but the implementation requires that the daemon be in a separate repo.